src/create-tasks.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  CommonErrorResponse,
+  ValidationErrorResponse,
+} from "@crowdin/crowdin-api-client";
+import {
+  CrowdinType,
+  createTask,
+  deleteTask,
+  isCommonErrorResponse,
+  listTasks,
+  unwrapErrorResponse,
+} from "./lib/crowdin";
+import log from "./utils/logging";
+
+export interface CreateTasksOptions {
+  branchName: string;
+  fileId: number;
+  languages: string[];
+  type: "proofread" | "translate";
+}
+
+const generateDescription = (branchName: string, language: string) =>
+  `DO NOT CHANGE: ${branchName} (${language})`;
+
+export default async (options: CreateTasksOptions) => {
+  log.info("Creating tasks...");
+
+  const tasks = await listTasks();
+  const descriptions = tasks.data.map(({ data: task }) => task.description);
+
+  for await (const language of options.languages) {
+    const title = `Review translations for ${options.branchName}`;
+    const description = generateDescription(options.branchName, language);
```

Comment:
@rppld I gave this some more thought, and I actually think we can do better. We only use the description to find the appropriate task, but tasks have file ids assigned to them, so I think we can just use those.

I was thinking this could work when deleting a branch. At the moment, we pass a branch name to `deleteBranch`, which then does a list branches and finds the branch ID. Instead, we could move this up one level like so:
1. `src/delete-branch.ts` takes a branch name, and finds the relevant branch ID.
2. We use Crowdin's [List Files](https://developer.crowdin.com/api/v2/#operation/api.projects.files.getMany) API to find the files for that branch ID. Will only be a single file btw.
3. In the loop that goes over all tasks, we match against the file ID instead of the description. If the `fileIds` array includes the file ID from our branch, we delete the task.

That way it's bulletproof and doesn't rely on description/label string matching.

What do you think?